sky/optimizer.py

```diff
@@ -772,8 +772,7 @@ def format_number(x):
 
         num_tasks = len(ordered_node_to_cost_map)
         for task, v in ordered_node_to_cost_map.items():
-            task_str = (f'for task_lib.Task {repr(task)!r}'
-                        if num_tasks > 1 else '')
+            task_str = (f'for task {repr(task)!r} ' if num_tasks > 1 else '')
```

Comment:
```suggestion
            task_str = (f'for task {repr(task)!r}' if num_tasks > 1 else '')
```
nit: seems like there is an extra space?

Reply:
Previous to this change, it looks like 

```
…Task ‘train’(1 node)
```